pkg/kubelet/kubelet_dockershim.go

```diff
@@ -58,10 +58,9 @@ func runDockershim(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 
 	// The unix socket for kubelet <-> dockershim communication, dockershim start before runtime service init.
-	klog.V(5).Infof("RemoteRuntimeEndpoint: %q, RemoteImageEndpoint: %q",
-		remoteRuntimeEndpoint,
-		remoteImageEndpoint)
-	klog.V(2).Infof("Starting the GRPC server for the docker CRI shim.")
+	klog.V(5).InfoS("RemoteRuntimeEndpoint and RemoteImageEndpoint", "remoteRuntimeEndpoint", remoteRuntimeEndpoint, "remoteImageEndpoint", remoteImageEndpoint)
```

Comment:
Remote is property of endpoint value and not 
```suggestion
	klog.V(5).InfoS("Using remote runtime endpoint and image endpoint", "runtimeEndpoint", remoteRuntimeEndpoint, "imageEndpoint", remoteImageEndpoint)
```